packages/visual-editor/src/store/tree.ts

```diff
@@ -64,6 +106,9 @@ export const useTreeStore = create<TreeStore>((set, get) => ({
 
     // The current and updated tree are the same, no tree update required.
     if (!treeDiff.length) {
+      console.debug(
+        `[exp-builder.visual-editor] updateTree(): During smart-diffing no diffs. Skipping tree update.`
```

Comment:
If we want to keep this log, I would like to align it with the other logs in this repo:
```suggestion
        `[exp-builder.visual-editor::updateTree] During smart-diffing no diffs. Skipping tree update.`
```